### ISSUE TITLE
[Failing Test] gce-cos-master-default (ci-kubernetes-e2e-gci-gce)

### ISSUE THREAD

**Issue by droslean** [kubernetes/kubernetes#85213]
**Which jobs are failing**:
`gce-cos-master-default (ci-kubernetes-e2e-gci-gce)`

**Since when has it been failing**:
`13th Nov 22:46 PST`

**Testgrid link**:
https://testgrid.k8s.io/sig-release-master-informing#gce-cos-master-default

**Reason for failure**:
```console
 W1113 13:46:56.575] scp: /var/log/kube-proxy.log*: No such file or directory
W1113 13:46:56.576] scp: /var/log/fluentd.log*: No such file or directory
W1113 13:46:56.577] scp: /var/log/node-problem-detector.log*: No such file or directory
W1113 13:46:56.577] scp: /var/log/kubelet.cov*: No such file or directory
W1113 13:46:56.577] scp: /var/log/startupscript.log*: No such file or directory
W1113 13:46:56.581] ERROR: (gcloud.compute.scp) [/usr/bin/scp] exited with return code [1].
W1113 13:46:57.011] scp: /var/log/kube-proxy.log*: No such file or directory
W1113 13:46:57.011] scp: /var/log/fluentd.log*: No such file or directory
W1113 13:46:57.011] scp: /var/log/node-problem-detector.log*: No such file or directory
W1113 13:46:57.011] scp: /var/log/kubelet.cov*: No such file or directory
W1113 13:46:57.011] scp: /var/log/startupscript.log*: No such file or directory
W1113 13:46:57.012] ERROR: (gcloud.compute.scp) [/usr/bin/scp] exited with return code [1].
W1113 13:46:57.403] scp: /var/log/kube-proxy.log*: No such file or directory
W1113 13:46:57.404] scp: /var/log/fluentd.log*: No such file or directory
W1113 13:46:57.404] scp: /var/log/node-problem-detector.log*: No such file or directory
W1113 13:46:57.404] scp: /var/log/kubelet.cov*: No such file or directory
W1113 13:46:57.404] scp: /var/log/startupscript.log*: No such file or directory
W1113 13:46:57.406] ERROR: (gcloud.compute.scp) [/usr/bin/scp] exited with return code [1].
W1113 13:46:57.577] scp: /var/log/kube-proxy.log*: No such file or directory
W1113 13:46:57.577] scp: /var/log/fluentd.log*: No such file or directory
W1113 13:46:57.577] scp: /var/log/node-problem-detector.log*: No such file or directory
W1113 13:46:57.577] scp: /var/log/kubelet.cov*: No such file or directory
W1113 13:46:57.578] scp: /var/log/startupscript.log*: No such file or directory
W1113 13:46:57.581] ERROR: (gcloud.compute.scp) [/usr/bin/scp] exited with return code [1].
W1113 13:47:00.749] INSTANCE_GROUPS=bootstrap-e2e-minion-group
W1113 13:47:00.750] NODE_NAMES=bootstrap-e2e-minion-group-7g14 bootstrap-e2e-minion-group-d8w2 bootstrap-e2e-minion-group-hnng bootstrap-e2e-minion-group-pnlw
I1113 13:47:01.731] Failures for bootstrap-e2e-minion-group (if any):
W1113 13:47:02.641] 2019/11/13 13:47:02 process.go:155: Step './cluster/log-dump/log-dump.sh /workspace/_artifacts' finished in 1m31.948444385s
W1113 13:47:02.642] 2019/11/13 13:47:02 process.go:153: Running: ./hack/e2e-internal/e2e-down.sh
W1113 13:53:40.750]   File "/workspace/./test-infra/jenkins/../scenarios/kubernetes_e2e.py", line 111, in check_env
W1113 13:53:40.750]     subprocess.check_call(cmd, env=env)
W1113 13:53:40.750]   File "/usr/lib/python2.7/subprocess.py", line 190, in check_call
W1113 13:53:40.750]     raise CalledProcessError(retcode, cmd)
W1113 13:53:40.751] subprocess.CalledProcessError: Command '('kubetest', '--dump=/workspace/_artifacts', '--gcp-service-account=/etc/service-account/service-account.json', '--up', '--down', '--test', '--provider=gce', '--cluster=bootstrap-e2e', '--gcp-network=bootstrap-e2e', '--check-leaked-resources', '--extract=ci/latest', '--gcp-master-image=gci', '--gcp-node-image=gci', '--gcp-nodes=4', '--gcp-zone=us-west1-b', '--ginkgo-parallel=30', '--publish=gs://kubernetes-release-dev/ci/latest-green.txt', '--test_args=--ginkgo.skip=\\[Slow\\]|\\[Serial\\]|\\[Disruptive\\]|\\[Flaky\\]|\\[Feature:.+\\] --minStartupPods=8', '--timeout=50m')' returned non-zero exit status 1
E1113 13:53:40.759] Command failed 
```

**Anything else we need to know**:
/cc @alenkacz @hasheddan @alejandrox1 
/sig cluster-lifecycle
/priority critical-urgent

**Comment by liggitt** [comment]
root cause is #76093 
fix in #85210 

see https://storage.googleapis.com/kubernetes-jenkins/logs/ci-kubernetes-e2e-gci-gce/1194516437415235587/artifacts/bootstrap-e2e-master/kube-master-installation.log:
```
Nov 13 07:30:49.463894 bootstrap-e2e-master configure.sh[733]: mv: cannot stat '/home/kubernetes/kubernetes/LICENSE': No such file or directory
Nov 13 07:30:49.464600 bootstrap-e2e-master systemd[1]: kube-master-installation.service: Main process exited, code=exited, status=1/FAILURE
Nov 13 07:30:49.464750 bootstrap-e2e-master systemd[1]: kube-master-installation.service: Failed with result 'exit-code'.```

**Comment by liggitt** [comment]
revert merged, job is green again
